### PR TITLE
Kazuha e causes fall dmg

### PR DESCRIPTION
**Pull request by itskhoi**

### CODE REVIEW

**Comment by Neptunya** [comment]
tbh I'm not sure if I should keep Basic Mechanics separate from Vault Entries or merge them together. I have an idea on how they can be merged together, I'll make a draft when I have more free time.

File: characters/anemo/kazuha.md [hunk 1]
```diff
@@ -72,6 +72,7 @@ Unleashes a secret technique as fierce as the rushing wind that pulls objects an
 * **Press:** Can be used in mid-air.
 
 * **Hold:** Charges up before unleashing greater Anemo DMG over a larger AoE than Press Mode. 
+  * Kazuha Hodling version of **Chihayaburu** can cause certain types of Enemies to take Fall DMG.
```
Comment (NoelleSimp) [inline characters/anemo/kazuha.md:75]: Holding

Comment (NoelleSimp) [reply characters/anemo/kazuha.md:75]: actually wowrding in general is awkward, probably just say something along the lines of 
"This version/variant can cause certain enemies to take Fall Damage"

Comment (itskhoi) [reply characters/anemo/kazuha.md]: fixed

**Pull request merged**